src/OpenTelemetry/CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ please check the latest changes
 ## Unreleased
 
 * Added `TracerProviderOptions` and `SetErrorStatusOnException`.
```

Comment:
`TracerProviderOptions` can be removed from changelog as well

Reply:
Updated.